dali/operators/image/crop/bbox_crop.cc

```diff
@@ -96,191 +96,232 @@ struct Range {
-of the bounding boxes with respect to the cropping window. Each sample will select one of the ``thresholds``
-randomly, and the operator will try a given number of attempts (see ``num_attempts``) to produce a random
-crop window that has the metric above the selected threshold.)code",
+        R"code(Minimum IoU or a different metric, if specified by ``threshold_type``, of the
+bounding boxes with respect to the cropping window.
+
+Each sample randomly selects one of the ``thresholds``, and the operator will complete
+the specified number of attempts (see ) to produce a random crop window that has
+the metric above the selected threshold. See ``num_attempts`` for more information about
+configurating the number of attempts.)code",
         std::vector<float>{0.f})
     .AddOptionalArg(
         "threshold_type",
-        R"code(Determines the meaning of ``thresholds``. By default refers to intersection-over-union (IoU) of
-the bounding boxes with respect to the cropping window. Alternatively, it could be set to ``overlap``
-to specify the fraction (by area) of the the bounding box which will fall inside the crop window
-(e.g. a threshold ``1.0`` means the whole bounding box must be contained in the resulting cropping window))code",
-        "iou")
+        R"code(Determines the meaning of ``thresholds``.
+
+By default, thresholds refers to the intersection-over-union (IoU) of the bounding boxes
+with respect to the cropping window. Alternatively, the threshold can be set to ``overlap`` to
+specify the fraction (by area) of the bounding box that will will fall inside the crop window.
+For example, a threshold value of ``1.0`` means the entire bounding box must be contained in the
+resulting cropping window.
+)code", "iou")
     .AddOptionalArg(
         "aspect_ratio",
-        R"code(Single or multiple valid aspect ratio ranges for cropping windows.
-
-For 2D bounding boxes, a single aspect ratio ``x/y`` range should be provided (i.e. ``[min_xy, max_xy]``)
-
-For 3D bounding boxes, three aspect ratio ranges are expected (``x/y``, ``x/z`` and ``y/z``) ranges, i.e
-``[min_xy, max_xy, min_xz, max_xz, min_yz, max_yz]``.
-Alternatively, if a single aspect ratio range is provided, the valid aspect ratio range will be the
-same for the three ratios.
+        R"code(Single or multiple valid aspect ratio ranges for the cropping windows.
 
-Value for ``min`` should be ``> 0.0`` and ``min <= max``.
+- Here some additional information: For 2D bounding boxes, one aspect ratio in the ``x/y`` range
+  should be provided (for example, ``[min_xy, max_xy]``).
+- | For 3D bounding boxes, the (``x/y``, ``x/z`` and ``y/z``) aspect ratio ranges are expected.
+  | An example is ``[min_xy, max_xy, min_xz, max_xz, min_yz, max_yz]``. Alternatively,
+    if an aspect ratio range is provided, the valid aspect ratio range will be the same for the
+    three ratios.
```

Comment:
This is not clear at all. The original documentation was very clear about when to provide what.
```suggestion
This parameter can be either a single value or a triple of values, depending on dimensionality of the input.
| For 2D bounding boxes, a single range of x/y aspect ratios should be provided (e.g. ``[min_xy, max_xy]``).
| For 3D bounding boxes, three separate aspect ratio ranges may be specified, for x/y, x/z and y/z pairs of
  dimensions. They are provided in the following order ``[min_xy, max_xy, min_xz, max_xz, min_yz, max_yz]``.
  Alternatively, if only one aspect ratio range is provided, the valid aspect ratio range will be the same for all
  three ratios.
```

Reply:
Done